server/src/modules/refresh-token/refresh-tokens.constants.ts

```diff
@@ -0,0 +1,10 @@
+import { ErrorMessagesEnum } from 'src/common/enums';
+import { TServiceErrorMessages } from 'src/common/types';
+
+export const refreshTokensServiceErrorMessages: TServiceErrorMessages = {
+  entitiesNotFound: ErrorMessagesEnum.REFRESH_TOKEN_NOT_FOUND,
+  entityNotFound: ErrorMessagesEnum.REFRESH_TOKEN_NOT_FOUND,
+  invalidData: ErrorMessagesEnum.INVALID_DATA,
+};
+
+export const SALT_ROUNDS = 10;
```

Comment:
move SALT_ROUND from users and refresh-tokens module to common/constants directory, bcs we use it in many different modules for hashing